src/data/content/modelli/comuni/adotta-il-modello-di-sito-comunale/realizza-l-interfaccia-del-sito-comunale.yaml

```diff
@@ -172,14 +172,26 @@ components:
   # SECTION EDITORIAL
   # ///////////////////
   sectionsEditorial :
-
-    - #title: I template HTML #C
+      
+    - title: Le librerie Figma #C
```

Comment:
@danielenole @ilascar @danielaiozzo @zetareticoli È un po' out of context che ci sia un brand di prodotto nel titolo di sezione. Si può cambiare in "Le librerie di componenti e layout di pagina"?

Reply:
sì io proporrei "Le librerie di componenti e layout"